fallout/smoke-test-fallout.yaml

```diff
@@ -0,0 +1,133 @@
+# region setup
+astra_provider: aws
+astra_region: eu-west-1
+
+# client setup
+client_count: 1
+client_provider: ec2
+client_region: eu-west-1
+client_type: m5.2xlarge
+client_nb_version: 5.17.3-release
+
+# keyspace setup
+keyspace: jsonapi_smoke_test
+
+# documents 10K to enable sorting
+docs_count: 10000
+
+# load setup
+op_cycles: 100000
+op_rate: 25
+op_connections: 3
+
+# ops from main block of smoke-test-nosqlbench.yaml
+ops:
+  - op: find-one-id
+  - op: find-one-by-city
+  - op: find-multi-by-married-name-exists
+  - op: find-multi-by-married-name-exists-with-projection
+  - op: find-multi-find-by-gender-sort-by-name
+  - op: find-by-name-newest-and-update-city
+  - op: delete-one-id
+  - op: find-one-id-and-replace-upsert
+  - op: count-all
+  - op: count-by-gender
+
+---
+
+ensemble:
+  server:
+    node.count: 1
+    provisioner:
+      name: astra
+      properties:
+        cloud.provider: {{astra_provider}}
+        cloud.region: {{astra_region}}
+        keyspace: {{keyspace}}
+        settle_time: 120
+
+  client:
+    node.count: {{client_count}}
+    provisioner:
+      name: ctool
+      properties:
+        cloud.provider: {{client_provider}}
+        cloud.region: {{client_region}}
+        cloud.instance.type: {{client_type}}
+    configuration_manager:
+      - name: nosqlbench_ssh
+        properties:
+          version: {{client_nb_version}}
+
+workload:
+  phases:
+
+    # set schema phase, ensures collection table is created from scratch
+    - nb_set_schema:
+        module: nosqlbench
+        properties:
+          client.size: 1
+          alias: schema
+          service_type: stargate
+          args:
+            - driver=http
+            - workload=https://raw.githubusercontent.com/stargate/jsonapi/ise-fallout/fallout/smoke-test-nosqlbench.yaml
```

Comment:
Should the file or branch be parametrized? In any way, it should point to `main` branch by default..